lib/runtime/wasmer/imports.go

```diff
@@ -875,8 +875,50 @@ func ext_trie_blake2_256_ordered_root_version_1(context unsafe.Pointer, dataSpan
 
 //export ext_trie_blake2_256_ordered_root_version_2
 func ext_trie_blake2_256_ordered_root_version_2(context unsafe.Pointer, dataSpan C.int64_t, version C.int32_t) C.int32_t {
-	// TODO: update to use state trie version 1 (#2418)
-	return ext_trie_blake2_256_ordered_root_version_1(context, dataSpan)
+	logger.Debug("executing...")
+
+	instanceContext := wasm.IntoInstanceContext(context)
+	memory := instanceContext.Memory().Data()
+	runtimeCtx := instanceContext.Data().(*runtime.Context)
+	data := asMemorySlice(instanceContext, dataSpan)
+
+	t := trie.NewEmptyTrie()
+	var values [][]byte
+	err := scale.Unmarshal(data, &values)
+	if err != nil {
+		logger.Errorf("[ext_trie_blake2_256_ordered_root_version_2]: %s", err)
```

Comment:
The function name is already logged with the wasmer logger automagically, see line

```go
logger.Patch(log.SetLevel(cfg.LogLvl), log.SetCallerFunc(true))
```

```suggestion
		logger.Errorf("%s", err)
```